src/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj

```diff
@@ -24,7 +24,7 @@
     <PackageReference Include="NuGet.ProjectModel" Version="$(CLI_NuGet_Version)" />
     <PackageReference Include="Microsoft.Build" Version="$(CLI_MSBuild_Version)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(CLI_MSBuild_Version)" />
-    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.0.0" />
+    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
```

Comment:
I think this change is fine, but in general this is the one library we ship publicly, so we need to be careful when we upgrade/edit dependencies.

Reply:
I'm aware. What we had was simply incorrect as we have indirect dependency on 4.3 already. We're likely causing downgrade warnings for consumers too.